app/webpacker/components/Panel/pages/FixResultsPage/api/getCompetitions.js

```diff
@@ -0,0 +1,17 @@
+import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../../../lib/requests/routes.js.erb';
+
+async function getCompetitions({ wcaId }) {
+  const { data } = await fetchJsonOrError(
+    viewUrls.persons.results(wcaId),
+  );
+  const resultsList = data || {};
+  const competitionsList = _.uniqBy(resultsList, 'competition_id')
+    .map((item) => ({
+      competitionId: item.competition_id,
+      competitionName: item.competition_name,
+    })).reverse();
```

Comment:
Just curious: What is the purpose of `reverse` here? If it's just a matter of sorting the dropdown options "more conveniently", then you should handle that outside of the data fetching layer.